### PR TITLE
erb files to slim

### PR DESCRIPTION
**Pull request by manishyadavv**
Currently there are some files that are still in ERB format , So they need to be converted to SLIM.
Converted Erb files in view folder in app to slim.

### CODE REVIEW

File: app/views/aliases/_index.html.erb [hunk 1]
```diff
@@ -1,2 +0,0 @@
-<%= render :partial => 'aliases/list', :locals => { :aliases => aliases } %>
```
Comment (vik-y) [inline app/views/aliases/_index.html.erb]: @manishyadavv Did you modify all these files using some script or manually?  What was the script/gem which you used?

**Comment by vik-y** [comment]
@manishyadavv I couldn't find where many of these partials were being used. But I used this gem "discover-unused-partials" and it says that the partials in aliases folder are being used. See the report below. I think there's a way to use those partials which we are not able to figure out maybe? : 

The following partials are not referenced directly by any code:
  app/views/aliases/_address_available.rjs
  app/views/aliases/_address_bad.rjs
  app/views/aliases/_name_available.rjs
  app/views/aliases/_name_bad.rjs
  app/views/aliases/_name_unavailable.rjs
  app/views/firewall/_clear_settings.rjs
  app/views/firewall/_fields_ip_filter.html.erb
  app/views/firewall/_fields_mac_filter.html.erb
  app/views/firewall/_fields_port_filter.html.erb
  app/views/firewall/_fields_port_forward.html.erb
  app/views/firewall/_fields_url_filter.html.erb
  app/views/firewall/_index.html.erb
  app/views/firewall/_ip_bad.rjs
  app/views/firewall/_ip_filter.rjs
  app/views/firewall/_ip_good.rjs
  app/views/firewall/_mac_bad.rjs
  app/views/firewall/_mac_filter.rjs
  app/views/firewall/_mac_good.rjs
  app/views/firewall/_port_filter.rjs
  app/views/firewall/_port_forward.rjs
  app/views/firewall/_port_range_bad.rjs
  app/views/firewall/_port_range_good.rjs
  app/views/firewall/_url_bad.rjs
  app/views/firewall/_url_filter.rjs
  app/views/firewall/_url_good.rjs
  app/views/hosts/_address_available.rjs
  app/views/hosts/_address_bad.rjs
  app/views/hosts/_address_unavailable.rjs
  app/views/hosts/_mac_bad.rjs
  app/views/hosts/_mac_good.rjs
  app/views/hosts/_mac_unavailable.rjs
  app/views/hosts/_name_available.rjs
  app/views/hosts/_name_bad.rjs
  app/views/hosts/_name_unavailable.rjs
  app/views/js_templates/shares/_extras_form.html.slim
  app/views/js_templates/shares/_gateway_form.html.slim
  app/views/js_templates/shares/_lease_time_form.html.slim
  app/views/js_templates/shares/_path_form.html.slim
  app/views/js_templates/shares/_tags_form.html.slim
  app/views/js_templates/shares/_workgroup_form.html.slim
  app/views/server/_one.html.erb
  app/views/shared/_errors.html.slim

  The following partials are only referenced directly by the partials above:
    app/views/firewall/_body.html.erb
    app/views/firewall/_new_button.html.erb

    The following partials are only referenced directly by the partials above:
      app/views/firewall/_list.html.erb
      app/views/firewall/_new.html.erb

      The following partials are only referenced directly by the partials above:
        app/views/firewall/_rule.html.erb
        app/views/firewall/_step1.html.erb

        The following partials are only referenced directly by the partials above:
          app/views/firewall/_details_ip_filter.html.erb
          app/views/firewall/_details_mac_filter.html.erb
          app/views/firewall/_details_port_filter.html.erb
          app/views/firewall/_details_port_forward.html.erb
          app/views/firewall/_details_url_filter.html.erb
          app/views/firewall/_empty_select.html.erb

**Comment by vik-y** [comment]
@cpg We are stuck with finding where a few partials are used. For example [_index.html.erb](app/views/aliases/_index.html.erb)

**Comment by manishyadavv** [comment]
Yes @cpg please tell us how can these be checked.

**Pull request merged**